src/ResourceManagement/Compute/Compute.Tests/SessionRecords/Compute.Tests.AvailabilitySetTests/TestOperations.json

```diff
@@ -1092,4 +1092,4 @@
   "Variables": {
```

Comment:
None of these changes that affect the line endings for compute json recordings should be there, pelase revery these